prow/config/external_plugins_config.yaml

```diff
@@ -588,9 +588,9 @@ ti-community-label:
       - 'good-first-issue'
       - 'hacktoberfest'
       - 'internal-test'
-      - 'needs-cherry-pick-4.0'
-      - 'needs-cherry-pick-5.0'
-      - 'needs-cherry-pick-5.0-rc'
+      - 'needs-cherry-pick-release-4.0'
+      - 'needs-cherry-pick-release-5.0'
+      - 'needs-cherry-pick-release-5.0-rc'
```

Comment:
We can remove this label now, branch `release-5.0-rc` is not maintained any more

Reply:
Got it. I will remove it.